docs/dockstore-logo-usage-guidelines.rst

```diff
@@ -0,0 +1,129 @@
+
+.. |Icon White| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-white.png
+  :width: 64
+
+===============================
+Dockstore Logo Usage Guidelines
+===============================
+
+About
+=====
+It is important to establish a sense of branding throughout our publicly facing, digital media. Currently, we have several themes / layouts, navigation systems, font sets, color palettes. We would like to unify our designs under a single set of standards.
+
+Color
+=====
+The official Dockstore color is:
+
+  .. list-table:: Dockstore Blue
+
+    * - |Dockstore Dot|
+      - Hex: #7986CB
+      - R: 121, G: 134, B: 203
+
+When using the Dockstore logo or icon, the acceptable colors are Dockstore Blue, black, or white as shown in the “Logo” section of this document.
+
+If the surrounding color is visually unappealing with Dockstore Blue, please use the black or white variants as appropriate.
+
+Logos
+=====
+
+  .. tabularcolumns:: |c|c|c|
+
+  .. list-table::
+    :header-rows: 1
+
+    * - Horizontal
+      - Vertical
+      - Icon
+    * - |Horizontal blue|
+      - |Vertical blue|
+      - |Icon blue|
+    * - Dockstore-logo-horizontal-blue
+      - Dockstore-logo-vertical-blue
+      - Dockstore-icon-blue
+    * - |Horizontal black|
+      - |Vertical black|
+      - |Icon black|
+    * - Dockstore-logo-horizontal-black
+      - Dockstore-logo-vertical-black
+      - Dockstore-icon-black
+    * - |Horizontal white|
+      - |Vertical white|
+      - |Icon white|
+    * - Dockstore-logo-horizontal-white
+      - Dockstore-logo-vertical-white
+      - Dockstore-icon-white
+    * - |Horizontal black blue|
+      - |Vertical black blue|
+      -
+    * - Dockstore-logo-horizontal-black-blue
+      - Dockstore-logo-vertical-black-blue
+      -
+    * - |Horizontal white blue|
+      - |Vertical white blue|
+      -
+    * - Dockstore-logo-horizontal-white-blue
+      - Dockstore-logo-vertical-white-blue
+      -
+
+A zip file containing these logos in both PNG and SVG formats can be found :download:`here </assets/images/Dockstore_Logos_and_Icons.zip>`.
+
+Use
+===
+Do:
+
+* Use the logo or icon color that works best in the context of the document, web page, or application.
+* The Dockstore logo is preferred.  Use the Dockstore icon only when space is limited or when it is clear that the reference is to Dockstore.
+* Spacing:
+  * In most cases, the logo should be of equal prominence to any other brands with which it is shown.
+  * Allow for adequate space around the logo when used in conjunction with other identifiers.
+  * Do not place other identifiers closer than the equivalent of the width of 1/2 the logo.
+
+Do Not:
+
+* Don’t place the logo on clashing colored backgrounds.
+* Don’t place the reverse logo on similar tints.
+* Don’t place any logo variation on busy backgrounds.
+* Don’t place special effects such as drop shadows on the logo.
+* Don’t skew or distort the logo.
+* Don’t alternate the color of the logo.
```

Comment:
Does this mean if you have two logos, they should have the same color? Or does it mean not change the color of the logo at all?

Reply:
Should read "Don't alter the color of the logo."  Thanks to Ash, this was changed in a subsequent push.

Reply:
The first paragraph has been reworked.